app/web/src/newhotness/ComponentGridTile.vue

```diff
@@ -1,20 +1,71 @@
 <template>
-  <div :class="clsx('component tile cursor-pointer')">
-    <header>
+  <div
+    :class="
+      clsx(
+        'component tile',
+        'cursor-pointer border border-neutral-500 rounded overflow-hidden',
+        themeClasses('bg-shade-0', 'bg-neutral-900'),
+      )
+    "
+  >
+    <header
+      :class="clsx('p-xs', themeClasses('bg-neutral-200', 'bg-neutral-800'))"
+    >
       <Icon name="logo-aws" size="lg" />
-      <h2>{{ component.name }}</h2>
-      <h3>{{ component.schemaName }}</h3>
-      <Icon name="loader" size="lg" />
+      <h2>
+        <TruncateWithTooltip class="pb-xs">{{
+          component.name
+        }}</TruncateWithTooltip>
+      </h2>
+      <h3>
+        <TruncateWithTooltip class="pb-xs">{{
+          component.schemaName
+        }}</TruncateWithTooltip>
+      </h3>
+      <Icon name="none" size="lg" />
     </header>
-    <ol>
-      <li>Qualifications: {{ component.qualificationTotals.total }}</li>
-      <li>Resource: {{ component.hasResource }}</li>
-      <li>Diff: {{ component.diffCount }}</li>
-      <hr />
-      <li>Inputs: {{ component.inputCount }}</li>
-      <li>Outputs: {{ component.outputCount }}</li>
+    <ol
+      class="[&>li]:p-xs [&>li]:flex [&>li]:flex-row [&>li]:items-center [&>li]:gap-xs [&_.pillcounter]:w-5 [&_.pillcounter]:h-5"
```

Comment:
Cool!

Reply:
Yes, very. Unfortunately tailwind is not at all interested in offering support (based on github issues) for collapsing multiple style applications like this into a single `[&>li]` statement... so... duplication it is!